api-types/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mattermost/desktop-api",
-  "version": "5.9.0-1",
+  "version": "5.9.0-2",
```

@streamer45 This should actually be v5.10 now
```suggestion
  "version": "5.10.0-1",
```

---

Too bad